features/bevragen/persoon/adressering/aanhef/dev/fields-gba.feature

```diff
@@ -17,17 +14,18 @@ Functionaliteit: Fields vertalen van proxy naar gba voor aanhef
       | geslachtsnaam (02.40)                | Hoogh     |
       | aanduiding naamgebruik (61.10)       | E         |
       En de persoon heeft een 'ex-partner' met de volgende gegevens
-      | naam                                                         | waarde         |
-      | voornamen (02.10)                                            | Anna Catharina |
-      | adellijke titel of predicaat (02.20)                         | BS             |
-      | voorvoegsel (02.30)                                          | van den        |
-      | geslachtsnaam (02.40)                                        | Aedel          |
-      | datum ontbinding huwelijk/geregistreerd partnerschap (07.10) | 20211109       |
+      | naam                                                               | waarde         |
+      | voornamen (02.10)                                                  | Anna Catharina |
+      | adellijke titel of predicaat (02.20)                               | BS             |
+      | voorvoegsel (02.30)                                                | van den        |
+      | geslachtsnaam (02.40)                                              | Aedel          |
+      | datum huwelijkssluiting/aangaan geregistreerd partnerschap (06.10) | 20010809       |
```

Comment:
hier wordt uitgegaan dat datum huwelijksluiting en datum ontbinding in dezelfde rij staan. Volgens https://github.com/BRP-API/Haal-Centraal-BRP-bevragen/blob/develop/features/bevragen/persoon/partner/aangaan-huwelijk-partnerschap/dev/aangaanHuwelijkPartnerschap-gba.feature zou dat toch niet voorkomen?

Reply:
klopt. Ik heb het aangepast (in de 3 features)